### PR TITLE
LF-4219: Create component for weight application rate tabs

### PR DESCRIPTION
**Pull request by SayakaOno**
**Description**

- Update `StateTab` component to
  - take `variant`.
  - replace `Semibold` with `button` so that tabs become focusable. (I plan to use `:focus-within` to style cards when focused, which will require this change. The outline is "none" for now.)
- Update `RoterTab` to be compatible with the new CSS.
- [Stories](http://localhost:6006/?path=/docs/components-routertab-statetabs--docs)

Jira link: https://lite-farm.atlassian.net/browse/LF-4219

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [x] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [x] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Comment by SayakaOno** [comment]
The design has changed, and this component is no longer necessary. Closing this pull request as it is no longer relevant.

**Comment by Duncan-Brain** [comment]
😢 Again!

File: packages/webapp/src/components/RouterTab/StateTab/index.jsx [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+ */
+
 import PropTypes from 'prop-types';
-import { Semibold } from '../../Typography';
 import styles from '../styles.module.scss';
 import clsx from 'clsx';
 
+const VARIANTS = {
+  PILL: 'pill',
+  PLANE: 'plane',
```
Comment (Duncan-Brain) [inline packages/webapp/src/components/RouterTab/StateTab/index.jsx:22]: "plain" 😄 not plane ✈️

**Review by Duncan-Brain — COMMENTED** [review]
Looks great! 

Only thing I noticed was when long text is squished the padding from the edge of the pill disappears. 
Screenshot 2024-06-04 at 3 37 41 PM

**Comment by kathyavini** [comment]
I get this funny bug in App on the exiting case of StateTabs only (not RouterTab); can you re-create it? It looks like a height bug on the non-focused tab?

Screenshot 2024-06-04 at 1 38 55 PM

Screenshot 2024-06-04 at 1 38 46 PM

File: packages/webapp/src/components/RouterTab/StateTab/index.jsx (hunk 1 above)
Comment (SayakaOno) [reply packages/webapp/src/components/RouterTab/StateTab/index.jsx]: 🙈🙈🙈

**Comment by SayakaOno** [comment]
Thank you for reviewing Duncan and Joyce!!!
I hope to reopen this again in the future 😂

**Pull request closed without merging**